source/includes/connect/tls-connection-str.c

```diff
@@ -0,0 +1,19 @@
+#include <mongoc/mongoc.h>
+#include <bson/bson.h>
+
+int main (void) {
+
+  // start-connect-to-atlas
+  // Initialize the MongoDB C Driver
+  mongoc_init ();
+
+  // Create a new client and set tls option to true
+  mongoc_client_t *client = mongoc_client_new ("mongodb+srv://<db_username>:<db_password>@<hostname/port>/?replicaSet=myRepl&tls=true");
```

Comment:
Q: do you need to set the `replicaSet` option here? if so, I'd describe that in the text; otherwise you can remove that option

Reply:
nope not needed!